File: abc/abc158/abc158_c.cs
// https://atcoder.jp/contests/abc158/submissions/10617339

using System;
using System.Linq;

namespace CSharp
{
    class Program
    {
        static void Main()
        {
            var input = Console.ReadLine().Split().Select(int.Parse).ToArray();
            var A = input[0];
            var B = input[1];

            for (int i = 0; i <= 1000; i++)
            {
                if (Math.Floor(i * 0.08) == A && Math.Floor(i * 0.1) == B)
                {
                    Console.WriteLine(i);
                    return;
                }
            }
            Console.WriteLine("-1");
        }
    }
}

File: abc/abc158/abc158_c.fs
// https://atcoder.jp/contests/abc158/submissions/11771789

let [| A; B |] = stdin.ReadLine().Split() |> Array.map int

let mutable result = 0

let rec solver k =
    if (k * 8 / 100) = A && (k * 10 / 100) = B then
        result <- k
        ()
    elif k = 1009 then ()
    else solver (k + 1)

solver 0
match result with
| 0 -> -1
| _ -> result
|> printfn "%d"
